contracts/utils/math/Math.sol

```diff
@@ -275,6 +275,33 @@ library Math {
         }
     }
 
+    /**
+     * @dev Returns the modular exponentiation of the specified base, exponent and modulus (b ** e % m)
+     *
+     * Requirements:
+     * - modulus can't be zero
+     * - result should be obtained successfully
+     */
```

Comment:
We should mention this function depends on the availability of the `0x05` precompile on the network where it's used as part of the assumptions of using this function.